README.md

```diff
@@ -1,32 +1,88 @@
-# Dev Container Templates: Self Authoring Guide
+# DevContainer Templates
+A collection of LocalStack related and managed DevContainer Templates.
 
-> This repo provides a starting point and example for creating your own custom [Dev Container Templates](https://containers.dev/implementors/templates), hosted for free on GitHub Container Registry.  The example in this repository follows the [Dev Container Template distribution specification](https://containers.dev/implementors/templates-distribution/).  
+>Development Container Templates are source files packaged together that encode configuration for a complete development environment.
+A Template can be used in a new or existing project, and a [supporting tool](https://containers.dev/supporting) will use the configuration from the Template to build a development container.
+
+>[!NOTE]
+> This repo follows the [**proposed** DevContainer Template distribution specification](https://containers.dev/implementors/templates-distribution/).
+
+>[!WARNING]
+> This repo currently a work in progress which might introduce breaking changes without notification.
+
+## List of templates
+- LocalStack Docker-in-Docker: setup a devcontainer with LocalStack running in an internal Docker service
+- LocalStack Docker-outside-of-Docker: setup a devcontainer communicating with a separate LocalStack instance running on the same Docker network
+
+## Usage
+
+To use either of the templates use a supported tool and choose one of them, then either add custom values to the options or keep the defaults.
+Alternatively, copy the contents of the desired template's `.devcontainer` folder and customize it as necessary.
+
+Both variation of the templates adds automatically the [official LocalStack DevContainer Feature](https://github.com/localstack/devcontainer-feature), which installs the CLI and by demand the most popular Local™ Tools.
+Currently this calls for a **Debian-based** DevContainer image.
```

Comment:
After given it some considerations, I came to the conclusion that it would be a better place for the recommendations section in the official LocalStack docs.